lib/resources/example_json/dstu2_examples_patient.rb

```diff
@@ -1,23 +1,17 @@
 module Cerner
   module Resources
 
-    DSTU2_PATIENT_1 ||= {
+    DSTU2_PATIENT_ENTRY ||= {
       "resourceType": "Patient",
-      "id": "9827347",
+      "id": "4596007",
```

Comment:
When I search for this id in SNDENG I get the patient "SMARTAPP, GrowthChartBMI"